packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -1766,8 +1769,11 @@ class _RetainedSizeColumn extends ColumnData<Reference> {
 
   @override
   String getDisplayValue(Reference dataObject) {
-    final value = getValue(dataObject);
-    return '$value';
+    return '--';
```

Comment:
instead of doing this, can we just remove the column from the table where it is used? That way we won't be taking up unnecessary space in the table for a value with no meaning '--'. Having '--' intentionally displayed everywhere also makes it seem like something is wrong with the tool or that we can't process the data correctly.